src/commands/member_services/print.js

```diff
@@ -0,0 +1,83 @@
+const Command = require('../Command');
+const { getPdfNumOfPages } = require('../util/getPdfNumOfPages');
+const {
+  validateDiscordID,
+  printerHealthCheck,
+  pushDiscordPDFToSqs,
+  editUserPagesPrinted
+} = require('../../APIFunctions/Print.js');
+
+module.exports = new Command({
+  name: 'print',
+  description: 'Print PDF file',
+  aliases: ['print'],
+  example: 's!print',
+  permissions: 'member',
+  category: 'member services',
+
+  // eslint-disable-next-line no-unused-vars
+  execute: async (message, args) => {
+    const { id } = message.author;
+
+    const reply = await message.channel.send('Authenticating Print Request.');
+
+    let {isValid, pagesPrinted} = await validateDiscordID( id );
+    if (!isValid) {
+      return reply.edit('Connect your discord account with ' +
+        'SCE web then try again!');
+    }
+
+    let url;
+    let timeout = false;
+    reply.edit('Check your DM');
+    await message.author.send('Please submit one file at a time!');
+    const filter = collected => collected.author.id === message.author.id;
+    await message.channel.awaitMessages(filter, {max: 1, time: 15000})
+      .then(collected => {
+        url = collected.first().attachments.first().url;
+      })
+      .catch(() => {
+        timeout = true;
+      });
```

Comment:
your combining callbacks and await here, i would instead just do
```suggestion
    try {
      const attachments = await message.channel.awaitMessages(filter, {max: 1, time: 15000})
      url = collected.first().attachments.first().url;
    } catch() {
      return reply.edit('Your time is up! Type "s!print"' +
      'command again to restart printing process!');
    }
```